### PR TITLE
Remove dashboard from the product-is distribution.

### PR DESCRIPTION
**Pull request by thanujalk**

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
             <class name="org.wso2.identity.integration.test.user.export.UserInfoExportTestCase" />
             <class name="org.wso2.identity.integration.test.consent.SelfSignUpConsentTest"/>
             <class name="org.wso2.identity.integration.test.oauth2.OIDCDiscoveryTestCase"/>
-            <class name="org.wso2.identity.integration.test.identity.governance.AdminForcedPasswordResetTestCase"/>
+            <!--<class name="org.wso2.identity.integration.test.identity.governance.AdminForcedPasswordResetTestCase"/>-->
```
Comment (darshanasbg) [inline modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml]: Have to fix test cases, rather commenting.

File: modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml [hunk 2]
```diff
@@ -70,7 +70,7 @@
             <class name="org.wso2.identity.integration.test.entitlement.EntitlementRestServiceTestCase"/>
             <class name="org.wso2.identity.integration.test.requestPathAuthenticator.RequestPathBasicAuthenticationSSOTest" />
             <class name="org.wso2.identity.integration.test.idp.mgt.IdentityProviderMgtServiceTestCase" />
-            <class name="org.wso2.identity.integration.test.dashboard.IDENTITY5811TestCase" />
+            <!--<class name="org.wso2.identity.integration.test.dashboard.IDENTITY5811TestCase" />-->
```
Comment (darshanasbg) [inline modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml]: Have to fix test cases, rather commenting.

File: modules/distribution/src/assembly/bin.xml [hunk 3]
```diff
@@ -1007,16 +1007,6 @@
             </source>
             <outputDirectory>wso2is-${pom.version}/repository/conf</outputDirectory>
         </file>
-
-        <file>
-            <source>
-                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/service-providers/sp_dashboard.xml
```
Comment (darshanasbg) [inline modules/distribution/src/assembly/bin.xml]: We should remove the `sp_dashboard.xml` from the original feature as well.

File: modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml (hunk 2 above)
Comment (thanujalk) [reply modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml]: These test cases are testing the dashboard...

File: modules/distribution/src/assembly/bin.xml (hunk 3 above)
Comment (thanujalk) [reply modules/distribution/src/assembly/bin.xml]: The feature is removed so no longer we have sp_dashbard.xml in the pack.

**Pull request merged**